js/src/wp-seo-post-scraper.js

```diff
@@ -67,7 +67,8 @@ var updateAdminBar = require( './ui/adminBar' ).update;
 			snippetCite: this.getDataFromInput( 'cite' ),
 			primaryCategory: this.getDataFromInput( 'primaryCategory' ),
 			searchUrl: wpseoPostScraperL10n.search_url,
-			postUrl: wpseoPostScraperL10n.post_edit_url
+			postUrl: wpseoPostScraperL10n.post_edit_url,
+			permalink: wpseoPostScraperL10n.base_url + this.getDataFromInput( 'url' )
```

Comment:
I think it is better to use the variable 'url' instead of calling the 'getDataFromInput' function again.